Bundle/WidgetMapBundle/Manager/WidgetMapManager.php

```diff
@@ -162,7 +160,13 @@ public function delete(View $view, Widget $widget)
             $view->addWidgetMap($replaceWidgetMap);
         }
 
+        //Move children for current WidgetMap View
         $this->moveChildren($view, $beforeChild, $afterChild, $originalParent, $originalPosition);
+
+        //Move children WidgetMap for children from other View
+        foreach ($widgetMap->getChildren() as $child) {
+            $this->moveWidgetMap($child->getView(), $child, $originalParent, $originalPosition);
```

Comment:
This cas seems be valid in other context than deletion. Would it be consistant to make the moveWidgetMap for the children views inside the moveChildren method ?